apps/backend/.env.example

```
@@ -7,4 +7,5 @@ PREVIEW_URL=
 S3_BUCKET=
 AWS_REGION=
 AWS_SECRET_ACCESS_KEY=
-AWS_ACCESS_KEY_ID=
\ No newline at end of file
+AWS_ACCESS_KEY_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_ACCESS_KEY_ID key should go before the AWS_REGION key

(UnorderedKey)